### ISSUE TITLE
custom keyword - transfer learning 

### ISSUE THREAD

**Issue by Aishaj** [ARM-software/ML-examples#69]
Hello, 

I want to use this trained model as a base model; then, I will retrain the model with a custom dataset. The word that I want to train the model on is not included in the original commands dataset. 
Will this transfer learning process work ? 
The number of utterances for my keyword is not big.

Thank you

**Comment by Burton2000** [comment]
Yes this should be possible, you might need to modify the training code a little bit to load the pre-trained checkpoint. Also when loading weights you should try passing in ```skip_mismatch=True``` to avoid TF throwing an error when loading the model weights as you will have changed the model structure if you are training for a different number of keywords.

If doing transfer learning I don't think you would need too many samples in your dataset to get decent results.
